pkg/services/ngalert/eval/eval.go

```diff
@@ -238,9 +238,11 @@ func buildDatasourceHeaders(ctx EvaluationContext) map[string]string {
 }
 
 // getExprRequest validates the condition, gets the datasource information and creates an expr.Request from it.
-func getExprRequest(ctx EvaluationContext, data []models.AlertQuery, dsCacheService datasources.CacheService) (*expr.Request, error) {
+func getExprRequest(ctx context.Context, data []models.AlertQuery, dsCacheService datasources.CacheService) (*expr.Request, error) {
+	user := GetSignedInUser(ctx)
```

Comment:
I think we should check for `nil` here, but you could argue that the existing code also assumed the context would always contain a non-nil user.